samples/notebooks/python/02-running-prompts-from-file.ipynb

```diff
@@ -75,7 +75,7 @@
       "source": [
         "Given a semantic function defined by these files, this is how to load and use a file based semantic function.\n",
         "\n",
-        "Load and configure the kernel, as usual, loading also the AI service settings defined in the [Setup notebook](0-AI-settings.ipynb):"
+        "Load and configure the kernel, as usual, loading also the AI service settings defined in the [Setup notebook](01-basic-loading-the-kernel.ipynb):"
```

Comment:
I think `00-getting-started.ipynb` might be more appropriate, thoughts @alexchaomander ?

Reply:
Yeah for loading the kernel with AI service settings, `00-getting-started.ipynb` is a better place to start

Reply:
done!